### ISSUE TITLE
Right shift on windows acts as one-way toggle

### ISSUE THREAD

**Issue by Jamminroot** [flutter/flutter#87344]
## Steps to Reproduce

Right shift seem to be handled in an incorrect manner, similar to the bug with CTRL (which led to duplicated inputs).

Flutter f4abaa0735eba4dfd8f33f73363911d63931fe03
Engine 844c29f42a614420b2205c178f22d30b43a8b0bb

1. Create new project
2. Add a TextField somewhere on the form:
```dart
          // horizontal).
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            TextField(
              decoration: InputDecoration(
                hintText: 'Test here',
              ),
            ),
            Text(
              'You have pushed the button this many times:',
            ),
            Text(
              '$_counter',
              style: Theme.of(context).textTheme.headline4,
            ),
          ],
```
3. Press right shift, now all input is stuck in "caps", until **Left** shift is pressed

**Expected results:** 

RShift works

**Actual results:**

RShift is not working

Your cpt obvious here, I guess? :)

<details>
  <summary>Logs</summary>

```
C:\Users\jamminroot\IdeaProjects\untitled>flutter run --verbose
[  +49 ms] executing: [C:\Tools\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +42 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] f4abaa0735eba4dfd8f33f73363911d63931fe03
[        ] executing: [C:\Tools\flutter/] git tag --points-at f4abaa0735eba4dfd8f33f73363911d63931fe03
[  +30 ms] Exit code 0 from: git tag --points-at f4abaa0735eba4dfd8f33f73363911d63931fe03
[        ] 2.2.3
[   +3 ms] executing: [C:\Tools\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +21 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [C:\Tools\flutter/] git ls-remote --get-url origin
[  +19 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +72 ms] executing: [C:\Tools\flutter/] git rev-parse --abbrev-ref HEAD
[  +20 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +58 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +35 ms] executing: C:\Users\jamminroot\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +25 ms] List of devices attached
[   +3 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +53 ms] Skipping pub get: version match.
[  +73 ms] Generating C:\Users\jamminroot\IdeaProjects\untitled\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[  +58 ms] Initializing file store
[   +6 ms] Skipping target: gen_localizations
[   +4 ms] complete
[   +3 ms] Launching lib\main.dart on Windows in debug mode...
[   +3 ms] C:\Tools\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\Tools\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root C:\Tools\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter
--debugger-module-names --experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill C:\Users\JAMMIN~1\AppData\Local\Temp\flutter_tools.fcde5e6d\flutter_tool.1df9eade\app.dill --packages C:\Users\jamminroot\IdeaProjects\untitled\.dart_tool\package_config.json
-Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill build\3c113a45063dc6628e68a4111abcacad.cache.dill.track.dill --enable-experiment=alternative-invalidation-strategy
[  +16 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[  +15 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[        ] [
             {
               "instanceId": "191a6e7e",
               "installDate": "2021-07-06T15:08:22Z",
               "installationName": "VisualStudio/16.10.3+31424.327",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community",
               "installationVersion": "16.10.31424.327",
               "productId": "Microsoft.VisualStudio.Product.Community",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Community 2019",
               "description": "Мощная интегрированная среда разработки, бесплатная для студентов, участников проектов с открытым кодом и отдельных пользователей.",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "releaseNotes": "https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-v16.10#16.10.3",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2021-07-06T15:08:22.0952054Z",
               "catalog": {
                 "buildBranch": "d16.10",
                 "buildVersion": "16.10.31424.327",
                 "id": "VisualStudio/16.10.3+31424.327",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.10.3",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "3",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.10.3+31424.327",
                 "requiredEngineVersion": "2.10.2174.31177"
               },
               "properties": {
                 "campaignId": "1405165975.1625582544",
                 "channelManifestId": "VisualStudio.16.Release/16.10.3+31424.327",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\setup.exe"
               }
             }
           ]
[   +2 ms] Building Windows application...
[   +8 ms] <- compile package:untitled/main.dart
[   +2 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S C:\Users\jamminroot\IdeaProjects\untitled\windows -B build\windows -G Visual Studio 16 2019
[  +19 ms] -- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.19043.
[  +16 ms] -- Configuring done
[  +39 ms] -- Generating done
[   +2 ms] -- Build files have been written to: C:/Users/jamminroot/IdeaProjects/untitled/build/windows
[   +4 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug --target INSTALL --verbose
[  +73 ms] Microsoft (R) Build Engine version 16.10.2+857e5a733 for .NET Framework
[        ] Copyright (C) Microsoft Corporation. All rights reserved.
[  +40 ms] Build started 30.07.2021 19:36:40.
[ +187 ms] Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[        ] Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\INSTALL.vcxproj" (1) is building "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +19 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +43 ms]   Checking Build System
[  +38 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[        ] Done Building Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\ZERO_CHECK.vcxproj" (default targets).
[  +77 ms] Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\INSTALL.vcxproj" (1) is building "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[   +1 ms] Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) on node 1 (default targets).
[        ] Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) is building "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[        ]   "The build of 'C:\Users\jamminroot\IdeaProjects\untitled\build\windows\CMakeFiles\ec147e46609d1afc9a9b1c191d40465c\flutter_assemble.rule' depends on 'C:\USERS\JAMMINROOT\IDEAPROJECTS\UNTITLED\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'C:\Users\jamminroot\IdeaProjects\untitled\build\windows\CMakeFiles\1f6b80295c5e3df4795a14e379562699\flutter_windows.dll.rule'. The items cannot be built in parallel."
[  +19 ms]   Generating C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/flutter_windows.dll, C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/flutter_export.h, C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/flutter_windows.h,
C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/flutter_messenger.h, C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/flutter_plugin_registrar.h, C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/flutter_texture_registrar.h,
C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc, C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc,
C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc, C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc,
C:/Users/jamminroot/IdeaProjects/untitled/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[ +617 ms]   [  +48 ms] executing: [C:\Tools\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +36 ms]   [  +53 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ]   [        ] f4abaa0735eba4dfd8f33f73363911d63931fe03
[        ]   [        ] executing: [C:\Tools\flutter/] git tag --points-at f4abaa0735eba4dfd8f33f73363911d63931fe03
[  +38 ms]   [  +38 ms] Exit code 0 from: git tag --points-at f4abaa0735eba4dfd8f33f73363911d63931fe03
[        ]   [        ] 2.2.3
[   +3 ms]   [   +3 ms] executing: [C:\Tools\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +28 ms]   [  +27 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ]   [        ] origin/stable
[        ]   [        ] executing: [C:\Tools\flutter/] git ls-remote --get-url origin
[  +25 ms]   [  +25 ms] Exit code 0 from: git ls-remote --get-url origin
[        ]   [        ] https://github.com/flutter/flutter.git
[  +40 ms]   [  +40 ms] executing: [C:\Tools\flutter/] git rev-parse --abbrev-ref HEAD
[  +28 ms]   [  +28 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ]   [        ] stable
[  +39 ms]   [  +39 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [   +1 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +64 ms]   [  +65 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ]   [   +1 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +21 ms]   [  +23 ms] Initializing file store
[  +13 ms]   [  +13 ms] Skipping target: gen_localizations
[   +4 ms]   [   +4 ms] unpack_windows: Starting due to {}
[ +127 ms]   [ +127 ms] kernel_snapshot: Starting due to {}
[   +1 ms]   [   +2 ms] unpack_windows: Complete
[+1940 ms]   [+1939 ms] C:\Tools\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev C:\Tools\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root C:\Tools\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --target=flutter
--no-print-incremental-dependencies -DFLUTTER_WEB_AUTO_DETECT=true -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --packages C:\Users\jamminroot\IdeaProjects\untitled\.dart_tool\package_config.json --output-dill
C:\Users\jamminroot\IdeaProjects\untitled\.dart_tool\flutter_build\3d290874239c470ca63b7304565e8ae8\app.dill --depfile C:\Users\jamminroot\IdeaProjects\untitled\.dart_tool\flutter_build\3d290874239c470ca63b7304565e8ae8\kernel_snapshot.d package:untitled/main.dart
[+5744 ms]   [+5745 ms] kernel_snapshot: Complete
[ +307 ms]   [ +307 ms] debug_bundle_windows_assets: Starting due to {}
[  +78 ms]   [  +78 ms] debug_bundle_windows_assets: Complete
[ +340 ms]   [ +340 ms] Persisting file store
[   +6 ms]   [   +6 ms] Done persisting file store
[   +3 ms]   [   +4 ms] build succeeded.
[   +7 ms]   [   +6 ms] "flutter assemble" took 8┬а680ms.
[ +263 ms]   [ +265 ms] ensureAnalyticsSent: 262ms
[        ]   [        ] Running shutdown hooks
[        ]   [        ] Shutdown hooks complete
[        ]   [        ] exiting with code 0
[  +49 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[        ] Done Building Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[   +3 ms] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_app.dir\Debug\flutter_.FD9A854B.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[        ] CustomBuild:
[        ]   All outputs are up-to-date.
[  +30 ms] ClCompile:
[        ]   All outputs are up-to-date.
[  +13 ms] Lib:
[        ]   All outputs are up-to-date.
[        ]   flutter_wrapper_app.vcxproj -> C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\Debug\flutter_wrapper_app.lib
[   +6 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.FD9A854B.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_app.dir\Debug\flutter_.FD9A854B.tlog\flutter_wrapper_app.lastbuildstate".
[        ] Done Building Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[  +28 ms] Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (6) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.92C696C8.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[        ] CustomBuild:
[        ]   All outputs are up-to-date.
[   +5 ms] ClCompile:
[        ]   All outputs are up-to-date.
[   +2 ms] Lib:
[        ]   All outputs are up-to-date.
[        ]   flutter_wrapper_plugin.vcxproj -> C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[   +4 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.92C696C8.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.92C696C8.tlog\flutter_wrapper_plugin.lastbuildstate".
[        ] Done Building Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[  +25 ms] Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\ALL_BUILD.vcxproj" (3) is building "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\runner\untitled.vcxproj" (7) on node 1 (default targets).
[        ] InitializeBuildStatus:
[        ]   Creating "untitled.dir\Debug\untitled.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[        ] CustomBuild:
[        ]   All outputs are up-to-date.
[  +14 ms] ClCompile:
[        ]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\bin\HostX64\x64\CL.exe /c /IC:\Users\jamminroot\IdeaProjects\untitled\windows /IC:\Users\jamminroot\IdeaProjects\untitled\windows\flutter\ephemeral
/IC:\Users\jamminroot\IdeaProjects\untitled\windows\flutter\ephemeral\cpp_client_wrapper\include /Zi /nologo /W4 /WX /diagnostics:column /Od /Ob0 /D WIN32 /D _WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /D _UNICODE /D UNICODE /Gm- /EHsc
/RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17 /Fo"untitled.dir\Debug\\" /Fd"untitled.dir\Debug\vc142.pdb" /external:env:EXTERNAL_INCLUDE /external:W4 /Gd /TP /wd4100 /errorReport:queue
C:\Users\jamminroot\IdeaProjects\untitled\windows\flutter\generated_plugin_registrant.cc C:\Users\jamminroot\IdeaProjects\untitled\windows\runner\flutter_window.cpp
[  +36 ms]   generated_plugin_registrant.cc
[ +269 ms]   flutter_window.cpp
[ +521 ms]   Generating Code...
[  +51 ms] ResourceCompile:
[        ]   All outputs are up-to-date.
[  +21 ms] Link:
[        ]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\Users\jamminroot\IdeaProjects\untitled\build\windows\runner\Debug\untitled.exe" /INCREMENTAL /ILK:"untitled.dir\Debug\untitled.ilk" /NOLOGO
..\flutter\Debug\flutter_wrapper_app.lib C:\Users\jamminroot\IdeaProjects\untitled\windows\flutter\ephemeral\flutter_windows.dll.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker'
uiAccess='false'" /manifest:embed /manifestinput:C:\Users\jamminroot\IdeaProjects\untitled\windows\runner\runner.exe.manifest /DEBUG /PDB:"C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/untitled.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT
/IMPLIB:"C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/untitled.lib" /MACHINE:X64  /machine:x64 untitled.dir\Debug\Runner.res
[   +1 ms]   untitled.dir\Debug\flutter_window.obj
[        ]   untitled.dir\Debug\main.obj
[        ]   untitled.dir\Debug\run_loop.obj
[        ]   untitled.dir\Debug\utils.obj
[        ]   untitled.dir\Debug\win32_window.obj
[        ]   untitled.dir\Debug\generated_plugin_registrant.obj
[  +88 ms]   untitled.vcxproj -> C:\Users\jamminroot\IdeaProjects\untitled\build\windows\runner\Debug\untitled.exe
[   +4 ms] FinalizeBuildStatus:
[        ]   Deleting file "untitled.dir\Debug\untitled.tlog\unsuccessfulbuild".
[        ]   Touching "untitled.dir\Debug\untitled.tlog\untitled.lastbuildstate".
[        ] Done Building Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\runner\untitled.vcxproj" (default targets).
[   +4 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +18 ms]   Building Custom Rule C:/Users/jamminroot/IdeaProjects/untitled/windows/CMakeLists.txt
[  +21 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[        ] Done Building Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +3 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +34 ms] PostBuildEvent:
[        ]   setlocal
[        ]   "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -DBUILD_TYPE=Debug -P cmake_install.cmake
[        ]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[  +24 ms]   -- Install configuration: "Debug"
[        ]   -- Up-to-date: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/icudtl.dat
[        ]   -- Up-to-date: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/flutter_windows.dll
[   +4 ms]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets
[        ]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[        ]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[        ]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/fonts
[        ]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +2 ms]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[  +58 ms]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/NOTICES
[   +1 ms]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/packages
[        ]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[        ]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[        ]   -- Installing: C:/Users/jamminroot/IdeaProjects/untitled/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[   +7 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[        ] Done Building Project "C:\Users\jamminroot\IdeaProjects\untitled\build\windows\INSTALL.vcxproj" (default targets).
[        ] Build succeeded.
[        ]     0 Warning(s)
[        ]     0 Error(s)
[        ] Time Elapsed 00:00:11.57
[  +15 ms] Building Windows application... (completed in 11,8s)
[ +160 ms] Observatory URL on device: http://127.0.0.1:59823/ankPNszBxAI=/
[   +3 ms] Caching compiled dill
[  +17 ms] Connecting to service protocol: http://127.0.0.1:59823/ankPNszBxAI=/
[  +97 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:59823/ankPNszBxAI=/.
[  +43 ms] DDS is listening at http://127.0.0.1:59826/Z5oNFRlCVwU=/.
[  +40 ms] Successfully connected to service protocol: http://127.0.0.1:59823/ankPNszBxAI=/
[  +12 ms] DevFS: Creating new filesystem on the device (null)
[  +16 ms] DevFS: Created new filesystem on the device (file:///C:/Users/JAMMIN~1/AppData/Local/Temp/untitledc0817f4e/untitled/)
[   +1 ms] Updating assets
[  +54 ms] Syncing files to device Windows...
[   +2 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +2 ms] <- recompile package:untitled/main.dart 0144a466-2247-430d-a6bb-28320e100b51
[        ] <- 0144a466-2247-430d-a6bb-28320e100b51
[  +53 ms] Updating files.
[        ] DevFS: Sync finished
[   +1 ms] Syncing files to device Windows... (completed in 60ms)
[        ] Synced 0.0MB.
[   +1 ms] <- accept
[   +8 ms] Connected to _flutterView/0x245337272d0.
[   +2 ms] Flutter run key commands.
[   +1 ms] r Hot reload. 
[   +1 ms] R Hot restart.
[        ] h Repeat this help message.
[        ] d Detach (terminate "flutter run" but leave application running).
[   +1 ms] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ]  Running with sound null safety 
[   +1 ms] An Observatory debugger and profiler on Windows is available at: http://127.0.0.1:59826/Z5oNFRlCVwU=/
[ +617 ms] The Flutter DevTools debugger and profiler on Windows is available at: http://127.0.0.1:9102?uri=http%3A%2F%2F127.0.0.1%3A59826%2FZ5oNFRlCVwU%3D%2F
[+16460 ms] Application finished.
```

```
Analyzing untitled...
No issues found! (ran in 0.9s)
```

```
:\Users\jamminroot\IdeaProjects\untitled>flutter doctor -v
[√] Flutter (Channel stable, 2.2.3, on Microsoft Windows [Version 10.0.19043.1110], locale ru-RU)
    • Flutter version 2.2.3 at C:\Tools\flutter
    • Framework revision f4abaa0735 (4 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[!] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at C:\Users\jamminroot\AppData\Local\Android\sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: C:\Program Files\Common Files\Oracle\Java\javapath\java.exe
    • Java version Java(TM) SE Runtime Environment (build 16.0.1+9-24)
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.10.3)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.10.31424.327
    • Windows 10 SDK version 10.0.18362.0

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions).

[√] IntelliJ IDEA Ultimate Edition (version 2021.1)
    • IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA 2021.1.3
    • Flutter plugin version 58.0.3
    • Dart plugin version 211.7727

[√] VS Code (version 1.58.2)
    • VS Code at C:\Users\jamminroot\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.24.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19043.1110]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 92.0.4515.107
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 92.0.902.55

! Doctor found issues in 2 categories.

```

</details>

Have a nice weekend o/

**Comment by stuartmorgan** [comment]
Duplicate of https://github.com/flutter/flutter/issues/75675
